src/http/ngx_http_upstream.c

```diff
@@ -6847,7 +6843,7 @@ ngx_http_upstream_rbtree_insert_value(ngx_rbtree_node_t *temp,
 }
 
 
-static ngx_http_upstream_srv_conf_t *
+ngx_http_upstream_srv_conf_t *
```

Comment:
may need to add a macro here, if ngx_http_upstream_rbtree_lookup is original functin of nginx.

Reply:
@chobits in 6799 line, tengine have already MACRO it.

rbtree is optimization of tengine

Reply:
![image](https://github.com/alibaba/tengine/assets/3404496/53276776-fe61-42c1-af0c-e36897ca264d)

Reply:
done